README.md

```diff
@@ -0,0 +1,20 @@
+# Docker Cleaner v0.4.0
+
+Small ruby script to remove old containers and old images.
```

Comment:
Can you specify what is "old", and where does it remove the containers and images from? Does the script remove them from builders, apps node, etc?

Reply:
docker-cleaner is agnostic from Scalingo @EtienneM it removes them from any host it's installed on and run manually or as a cron task

Reply:
@EtienneM Is that OK now?

Reply:
Oh I didn't see it's a public project